assets/js/aboutCodeDataTables.js

```diff
@@ -245,14 +245,24 @@ class AboutCodeDataTable {
                         .map((column) => `${column.name}:name`)
                 },
                 {
+                    // Show only package columns
```

Comment:
Could you change the order of the buttons to be in the following order:

1. Column Visibility
2. Show all
3. Hide all
4. Origin info
5. Copyright info
6. License info
7. Package info